src/mixins/enum-mixin.ts

```diff
@@ -227,9 +227,10 @@ export default class EnumMixin extends Vue {
    * @param type the dissolution type to convert
    * @returns the dissolution name
    */
-  dissolutionTypeToName (type: DissolutionTypes): string {
+  dissolutionTypeToName (type: DissolutionTypes, isFirm: boolean = false): string {
     switch (type) {
-      case DissolutionTypes.VOLUNTARY: return DissolutionNames.VOLUNTARY
+      case DissolutionTypes.VOLUNTARY:
+        return isFirm ? DissolutionNames.FIRM_DISSOLUTION : DissolutionNames.VOLUNTARY
```

Comment:
Ideally there would be a new dissolution type for "firm voluntary", but I guess the API doesn't do that. So I'm OK with having this special case here.

However, I think you should import the `isFirm` getter in this mixin instead of passing it in as an optional parameter. I think it will be just a bit cleaner.

Reply:
@meganawong Please let the author resolve their comments. That way they know to check back and see if the code change is good. (Otherwise reviewers need to review all the code again.)